superset/utils/core.py

```diff
@@ -928,9 +928,12 @@ def send_email_smtp(  # pylint: disable=invalid-name,too-many-arguments,too-many
     # Attach any inline images, which may be required for display in
     # HTML content (inline)
     for msgid, imgdata in (images or {}).items():
-        image = MIMEImage(imgdata)
+        formatted_time = formatdate(localtime=True)
+        file_name = f"{subject} {formatted_time}"
+        image = MIMEImage(imgdata, name=file_name)
         image.add_header("Content-ID", "<%s>" % msgid)
         image.add_header("Content-Disposition", "inline")
+        image.add_header("Content-Disposition", 'attachment; filename="%s"' % file_name)
```

Comment:
what's the difference between this line, and what we're doing in line 937?

Reply:
good question, I am running it now without the file in image to see. The documentation used both so I emulated and then wasn't able to properly test.

Reply:
Ok, I tested, and the add_header lines seems inconsequential to the process that we want, so I deleted it.